src/services/prismic.js

```diff
@@ -54,13 +51,8 @@ export function queryDocumentsByType<T>(type: string, options?: QueryOptions): R
   return prismicClient.query(Prismic.Predicates.at(DOCUMENT_TYPE, type), options);
 }
 
-export async function queryDocumentsByID(id: string): Promise<string> {
-  const fetchPrismicDocumentResponse = await prismicClient.getByID(id);
-  const document = fetchPrismicDocumentResponse.data;
-  const prismicContent = [];
-  mapFromDocumentDataToString(document?.title, prismicContent);
-  mapFromDocumentDataToString(document?.subtitle, prismicContent);
-  mapFromDocumentDataToString(document?.content, prismicContent);
-  const prismicConvertedHTMLData = prismicContent.join('');
-  return prismicConvertedHTMLData;
+export async function queryDocumentsByID(id: string): Promise<Object> {
```

Comment:
Is Object `Prismic.DocumentData` or else?

```suggestion
export async function queryDocumentsByID(id: string): Promise<Object> {
```

Reply:
@poocart It is of generic type.